app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,12 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all' %>
+    <noscript>
+      <style>
+        .proceeding-item { display: block; }
+        #search-field { display: none; }
+      </style>
+    </noscript>
```

Comment:
It needs to be in the `<head>` tag, I couldn't see how to do that and make it only appear in the proceeding types page.  Adding in extra code for this seemed overkill - since we pull in all the other CSS on all pages, whether it is used or not.